appveyor.yml

```diff
@@ -31,39 +43,77 @@ install:
   - git clone --quiet --depth 1 https://github.com/groonga/groonga-normalizer-mysql.git storage\mroonga\vendor\groonga\vendor\plugins\groonga-normalizer-mysql
-build_script:
-  - "echo # > win\\packaging\\CMakeLists.txt"
-  - cmake . -G "%CMAKE_GENERATOR_NAME%"
-      -DCMAKE_BUILD_TYPE=Debug
-      -DPLUGIN_ARCHIVE=NO
-      -DPLUGIN_BLACKHOLE=NO
-      -DPLUGIN_CASSANDRA=NO
-      -DPLUGIN_CONNECT=NO
-      -DPLUGIN_CSV=NO
-      -DPLUGIN_EXAMPLE=NO
-      -DPLUGIN_FEDERATED=NO
-      -DPLUGIN_FEDERATEDX=NO
-      -DPLUGIN_HEAP=NO
-      -DPLUGIN_INNOBASE=YES
-      -DPLUGIN_MYISAM=NO
-      -DPLUGIN_MYISAMMRG=NO
-      -DPLUGIN_OQGRAPH=NO
-      -DPLUGIN_PERFSCHEMA=NO
-      -DPLUGIN_SEQUENCE=NO
-      -DPLUGIN_SPHINX=NO
-      -DPLUGIN_SPIDER=NO
-      -DPLUGIN_TEST_SQL_DISCOVERY=NO
-      -DPLUGIN_TOKUDB=NO
-      -DPLUGIN_XTRADB=NO
-      -DWITH_UNIT_TESTS=OFF
-      -DWITH_MARIABACKUP=OFF
-      -DGRN_WITH_BUNDLED_MECAB=ON
-  - cmake --build . --config Debug
-
 test: off
+
+for:
+  -
+    matrix:
+      only:
+        - configuration: Debug
+    build_script:
+      - "echo # > win\\packaging\\CMakeLists.txt"
+      - cmake . -G "%CMAKE_GENERATOR_NAME%"
+          -DCMAKE_BUILD_TYPE=Debug
+          -DPLUGIN_ARCHIVE=NO
+          -DPLUGIN_BLACKHOLE=NO
+          -DPLUGIN_CASSANDRA=NO
+          -DPLUGIN_CONNECT=NO
+          -DPLUGIN_CSV=NO
+          -DPLUGIN_EXAMPLE=NO
+          -DPLUGIN_FEDERATED=NO
+          -DPLUGIN_FEDERATEDX=NO
+          -DPLUGIN_HEAP=NO
+          -DPLUGIN_INNOBASE=YES
+          -DPLUGIN_MYISAM=NO
+          -DPLUGIN_MYISAMMRG=NO
+          -DPLUGIN_OQGRAPH=NO
+          -DPLUGIN_PERFSCHEMA=NO
+          -DPLUGIN_SEQUENCE=NO
+          -DPLUGIN_SPHINX=NO
+          -DPLUGIN_SPIDER=NO
+          -DPLUGIN_TEST_SQL_DISCOVERY=NO
+          -DPLUGIN_TOKUDB=NO
+          -DPLUGIN_XTRADB=NO
+          -DWITH_UNIT_TESTS=OFF
+          -DWITH_MARIABACKUP=OFF
+          -DGRN_WITH_BUNDLED_MECAB=ON
+      - cmake --build . --config Debug
+  -
+    matrix:
+      only:
+        - configuration: Package
+    environment:
+      MROONGA_VERSION: 9.00
+    build_script:
+      - set
+      - cd %APPVEYOR_BUILD_FOLDER%\..
+      - if "%CMAKE_GENERATOR_NAME%" == "Visual Studio 14 2015 Win64" if "%MARIADB_VERSION%" == "10.3.12" set BUILD_PACKAGE=TRUE
+      - if not %BUILD_PACKAGE% == "TRUE" exit 0
+      - git clone --depth 1
+        https://github.com/cosmo0920/PowerShell-for-Mroonga-building.git mrnbuild
+      - cd mrnbuild/powershell
+      - set "PATH=C:\msys64\usr\bin;C:\msys64\mingw64\bin;C:\Windows\System32;C:\Windows;%PATH%"
+      - set MSYSTEM=MINGW64
+      - pacman -Sy --noconfirm bison
+      - ps: .\get-mroonga-batfiles.ps1
+      - ps: .\prepare-building-mroonga.ps1
+      - cmake --version
+      - bison --version
+      - move ..\..\mariadb-%MARIADB_VERSION% work/source
+      - cd .\work
+      - call .\build-vc2015-zip-64.bat
+      - cd ..\
+      - ps: .\package-unzip.ps1
```

Comment:
Could you use `7z x` for other archive extractor?
Currently, this posershell script loads configuration which specifies other MariaDB version.
So, it causes CI failure.

Reply:
Fixed